### ISSUE TITLE
FYI - Softlayer maintenance, ongoing this week (May 2-7)

### ISSUE THREAD

**Issue by mhdawson** [nodejs/build#406]
Just an FYI - no expected impact

Customer Identification: SoftLayer Internal - Node.js Community Account (729957)
Start Date: Monday 02-May-2016 14:00 UTC
End Date: Sunday 08-May-2016 02:00 UTC
Duration: 5 days
Event Type: Planned Event
Subject: Event 27321405 - Critical Power Maintenance - SJC01.SR01/SR02

=================================================================
/ Event Description /
Critical power maintenance will be performed at our SJC01 facility through the days of Monday, May 2nd through Saturday, May 7th. This is a continuation of the postponed maintenance from early February. This maintenance will take place between the hours of 7am and 7pm PDT, there is no planned impact to servers or services during this time. In the event of an unexpected impact, we will work with our vendors to bring critical services back up as quickly as possible. 

The maintenance schedule is as follows: 
May 2nd From 7am to 7pm PDT, effected racks are: 243-264, 309-330, 375-396 in SJC01.SR02
May 3rd From 7am to 7pm PDT, effected racks are: 243-264, 309-330, 375-396 in SJC01.SR02
May 4th From 7am to 7pm PDT, effected racks are: 199-242, 265-286, 331-374 in SJC01.SR02
May 5th From 7am to 7pm PDT, effected racks are: 199-242, 265-286, 331-374 in SJC01.SR02
May 6th From 7am to 7pm PDT, effected racks are: 1-33, 67-88, 133-154, 175, 176 in SJC01.SR01
May 7th From 7am to 7pm PDT, effected racks are: 1-33, 67-88, 133-154, 175, 176 in SJC01.SR01

/ Items Associated With This Event /
Item ID | Hostname | Public IP | Private IP | Item Type
186873 | nodejsb1.softlayer.com | 50.23.85.254 | 10.52.6.135 | Bare Metal Server
13762115 | test-softlayer-centos5-x64-1.softlayer.com | 50.23.85.252 | 10.52.6.146 | Virtual Server
13762231 | release-softlayer-centos5-x86-1.softlayer.com | 50.23.85.253 | 10.52.6.148 | Virtual Server
13762233 | test-softlayer-centos6-x64-1.softlayer.com | 50.23.85.251 | 10.52.6.158 | Virtual Server
13762235 | test-softlayer-centos7-x64-1.softlayer.com | 50.23.85.250 | 10.52.6.133 | Virtual Server
13895573 | nodejs-data2.softlayer.com | 50.97.245.4 | 10.52.6.151 | Virtual Server
14113917 | test-softlayer-ubuntu1404-x64-1.softlayer.com | 50.97.245.5 | 10.52.6.156 | Virtual Server
14113919 | test-softlayer-ubuntu1404-x86-1.softlayer.com | 50.97.245.9 | 10.52.6.157 | Virtual Server

/ Event Updates /
- As of Tuesday 03-May-2016 01:48 UTC
Maintenance for May 2nd has been completed successfully, all effected racks are in normal service now. This maintenance will continue, on schedule, tomorrow at 7:00AM PDT

**Comment by jbergstroem** [comment]
TL;DR "no planned impact"

**Comment by jbergstroem** [comment]
Closed.
